Entities/Items/Crate/Crate.as

```diff
@@ -723,7 +745,7 @@ CBlob@ getPlayerInside(CBlob@ this)
 	return null;
 }
 
-bool DumpOutItems(CBlob@ this, float pop_out_speed = 5.0f, Vec2f init_velocity = Vec2f_zero, bool dump_player = true)
+bool DumpOutItems(CBlob@ this, float pop_out_speed = 5.0f, Vec2f init_velocity = Vec2f_zero, bool dump_player = true, bool dump_mine = true)
```

Comment:
Instead of having both `dump_player` and `dump_mine`, I suggest you just have one called `dump_special`. In most, if not all cases this method is used, you either want to dump out everything or dump out everything except players and mines. This way it allows more blobs to easily be added if needed in the future.